pipelines/nextflow/workflows/dumper_pipeline/main.nf

```diff
@@ -14,6 +14,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// dealing with custom $NXF_WORK based workDir
+//   don't do anything if "-work-dir (-w)" option specified on command line
+include { updateWorkDirAsNeeded } from '../../modules/utils/utils.nf'
+updateWorkDirAsNeeded("dumper_pipeline2")
+
+
+// default params
+params.help = false
+params.prefix = ''
+params.brc_mode = 0
+params.dbname_re = ''
+params.output_dir = './dumper_output'
+params.password = ''
+
+// Print usage
+def helpMessage() {
+  log.info '''
+        Usage:
+        The typical command for running the 'Dumper' pipeline is as follows:
+
+        CMD=<dba_alias>
```

Comment:
I would be inclined to leave the detail run cmd description to the readme and only have actual NXF runtime params described as part of main.nf.

Reply:
I'd rather have a full "use case" here. Though, we can discuss it

Reply:
changed to:
```
        nextflow run \
            -w ${data_dir}/nextflow_work \
            ensembl-genomio/pipelines/nextflow/workflows/dumper_pipeline/main.nf \
            -profile lsf \
            --host <DB_HOST> --port <DB_PORT> --user <DB_USER>
            --dbname_re '^drosophila_melanogaster_\w+_57_.*$' \
            --output_dir ${data_dir}/dumper_output
```